code/game/objects/items/stunbaton.dm

```diff
@@ -164,6 +164,20 @@
 			to_chat(user, "<span class='warning'>[src] is out of charge.</span>")
 	update_icon()
 	add_fingerprint(user)
+	
+/obj/item/melee/baton/proc/move_to_jail(mob/living/L, user)
+	//pick randomly from a list of prison turfs to teleport the RDMer to
+	var/list/validturfs = list() //Find all open prison turfs so we can pick from them later
+	for(var/area/security/prison/P in SSmapping.areas_in_z["[SSmapping.station_start]"])
+		for(var/turf/open/floor/F in P.contents)
+			validturfs += F
+	if(LAZYLEN(validturfs))
+		var/turf/newturf = pick(validturfs)
+		L.loc = newturf
+		priority_announce("[L] has been arrested for 300 seconds")
+		to_chat(L,"You have been arrested for 300 seconds")

```

Comment:
come on bro start a timer to teleport them back to their job spawn point after 300 seconds, shouldn't be too hard

Reply:
I've pushed this change but I think the pr has to be open for it to appear here. 
I couldn't send them to jobspawns bc they get deleted so I just tp them to arrivals or hallway if theres no valid turfs

Reply:
@optimumtact should he make a new PR and manually close that one, or should he ask a maintainer to re-open this one, because the testmerge isn't going to get the updated version unless the PR is opened so github actually registers the new commit

we've tried testmerging closed stuff before, I tried telling you this in the coding channel but you ignored me lol

Reply:
I ignored you because I don't care, frankly I think most of you have no chance of getting a test merge, and I will not bother myself with this issue further